llvm/include/llvm/Transforms/Vectorize/LoopVectorizationLegality.h

```diff
@@ -446,6 +480,9 @@ class LoopVectorizationLegality {
   /// specific checks for outer loop vectorization.
   bool canVectorizeOuterLoop();
 
+  /// Returns true if this is an early exit loop that can be vectorized.
```

Comment:
It would probably be helpful to summarize the required conditions here (or somewhere else)

Reply:
Done